infrastructure/builder/src/generate/generators/tc-client-go.js

```diff
@@ -4,12 +4,10 @@ const exec = util.promisify(require('child_process').execFile);
 const {REPO_ROOT, readRepoFile, execCommand} = require('../../utils');
 
 exports.tasks = [{
-  title: 'Generate Taskcluster-Client-Go',
-  requires: ['references-json'],
-  provides: ['target-taskcluster-client-go'],
+  title: 'Check Go Version',
```

Comment:
(Not related to this PR, but I wonder if this would make a useful meta-test for the repo?)

Reply:
Maybe, but it's a test of the environment (what version of go is installed) rather than of the source code.